llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -379,7 +403,19 @@ async def _acreate_index(self, index_name: Optional[str]) -> None:
         logger.info(f"Configuring {index_name} metadata fields")
         metadata_index_fields = self._create_metadata_index_fields()
         fields.extend(metadata_index_fields)
-        logger.info(f"Configuring {index_name} vector search")
+        # Determine the compression type
+        compressions = []
+        if self._compression_type == "binary":
```

Comment:
Same comment as for the sync version. btw - the sync and async versions of this method do the same setup of index definition, we could have that in a single method and then call it from both instead of having a copy of this code.